sdk/communication/communication-short-codes/review/communication-short-codes.api.md

```diff
@@ -0,0 +1,212 @@
+
+// @public (undocumented)
+export interface MessageDetails {
+    // (undocumented)
+    confirmationMessage?: string;
+    directionality?: MessageDirectionality;
+    helpMessage?: string;
+    // (undocumented)
+    optInMessage?: string;
+    optInReply?: string;
+    optOutMessage?: string;
+    recurrence?: Recurrence;
+    supportedProtocols?: MessageProtocol[];
+    useCases?: UseCase[];
+}
+
+// @public
+export type MessageDirection = "toUser" | "fromUser";
+
+// @public
+export type MessageDirectionality = "oneWay" | "twoWay";
+
+// @public
+export interface MessageExample {
+    direction?: MessageDirection;
+    text?: string;
+}
+
+// @public
+export interface MessageExampleSequence {
+    messages?: MessageExample[];
+}
+
+// @public
+export type MessageProtocol = "sms" | "mms";
+
+// @public
+export type NumberType = "shortCode" | "alphaId";
+
+// @public
+export type ProgramBriefStatus = "submitted" | "approved" | "submitNewVanityNumbers" | "updateProgramBrief" | "draft" | "denied";
+
+// @public (undocumented)
+export interface ProgramDetails {
+    description?: string;
+    expectedDateOfService?: Date;
+    isPoliticalCampaign?: boolean;
+    isVanity?: boolean;
+    name?: string;
+    numberType?: NumberType;
+    preferredVanityNumbers?: string[];
+    privacyPolicyUrl?: string;
+    signUpTypes?: ProgramSignUpType[];
+    signUpUrl?: string;
+    termsOfServiceUrl?: string;
+    url?: string;
+}
+
+// @public
+export type ProgramSignUpType = "website" | "pointOfSale" | "sms" | "interactiveVoiceResponse";
+
+// @public
+export type Recurrence = "subscription" | "transaction";
+
+// @public
+export interface ReviewNote {
+    date?: Date;
+    message?: string;
+}
+
+// @public
+export interface ShortCode {
+    countryCode?: string;
+    number?: string;
+    numberType?: NumberType;
+    programBriefIds?: string[];
+    purchaseDate?: Date;
+}
+
+// @public
+export interface ShortCodeCost {
+    amount: number;
+    billingFrequency: BillingFrequency;
+    currencyCode: string;
+}
+
+// @public (undocumented)
+export class ShortCodesClient {
+    constructor(connectionString: string, options?: ShortCodesClientOptions);
+    constructor(url: string, credential: KeyCredential, options?: ShortCodesClientOptions);
+    constructor(url: string, credential: TokenCredential, options?: ShortCodesClientOptions);
+    // (undocumented)
+    deleteUSProgramBrief(programBriefId: string, options?: DeleteUSProgramBriefOptions): Promise<RestResponse>;
```

Comment:
Curious how this client API surface will grow once we support a gazillion countries ;-)